core/pom.xml

```diff
@@ -129,5 +129,11 @@
             <version>1.0.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
As I remember uber-jar dependency already has apache commons so you don't need to add apache commons by yourself